src/js/pages/catalog/PackageDetailTab.js

```diff
@@ -310,18 +287,28 @@ class PackageDetailTab extends mixin(StoreMixin) {
   getPackageVersionsDropdown() {
     const cosmosPackage = CosmosPackagesStore.getPackageDetails();
     const packageName = cosmosPackage.getName();
-    const cosmosPackages = CosmosPackagesStore.getPackages();
+    const cosmosPackagesVersions = CosmosPackagesStore.getPackagesVersions();
```

Comment:
It is possible for the `cosmosPackagesVersions` to be null and it's accessed later without checking or handling this.

Reply:
I don't understand what you trying to say here.

Reply:
Talked offline: will not return `UniversePackagesVersions({})` from `getPackagesVersions()` when packagesVersions field is undefined.